src/redux/actions/group-actions.js

```diff
@@ -174,6 +174,7 @@ export const removeMembersFromGroup = (groupId, members) => {
 export const fetchRolesForGroup = (groupId, pagination, options) => ({
```

Comment:
There I would either:
- add a param that would switch between fetching roles for a group detail and expandable table in groups
- or have separate action for fetching the data for group detail (stored as selectedGroup in redux) and expanded Groups rows (stored in groups entries)

Reply:
why do I need to separate the data, even though it is the same data in each group detail and expandable table?

Reply:
yeah, it's the same data but you probably don't want to store them in `selectedGroup` object in store but rather enhance given group in the `groups.data` array. Otherwise, you would lose previously fetched data on every expand - new group detail data would overwrite the old one.